Sources/Build/misc.swift

```diff
@@ -77,9 +78,9 @@ extension ClangModule {
         
         let umbrellaHeader = Path.join(moduleHeaderDir, "\(name).h")
```

Comment:
I'm tempted to say `c99name` is the right thing to look for here, given that is what users will write in source code. I think we should start with that, and if `c99name` != `name` then produce a warning if `name.h` exists.